### PR TITLE
13175 public websites react component migration part 2

### PR DESCRIPTION
**Pull request by jtmst**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- Migrates to web-components for design system components for code in the public websites portfolio.  This includes alertBox, additional-info, loading indicator, and select components

## Related issue(s)

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13175

## Testing done

- local testing in components where they show up
- Will need to test CTA component changes in review instance

## Screenshots
Screen Shot 2023-04-24 at 12 49 16 PM
Screen Shot 2023-04-25 at 2 44 57 PM
Screen Shot 2023-04-25 at 4 21 37 PM
Screen Shot 2023-04-25 at 12 53 38 PM

## What areas of the site does it impact?

*(Describe what parts of the site are impacted and*if*code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.go header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx [hunk 1]
```diff
@@ -98,11 +97,10 @@ const ResourcesAndSupportSearchApp = () => {
     <div className="usa-grid usa-grid-full">
       <div className="usa-content vads-u-margin-bottom--0 medium-screen:vads-u-margin-bottom--3">
         {errorMessage && (
-          <AlertBox
-            headline="Something went wrong"
-            status="error"
-            content={errorMessage}
-          />
+          <va-alert status="error">
+            <h3 slot="headline">"Something went wrong"</h3>
```
Comment (randimays) [inline src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx:101]: `Something went wrong` probably should not have quotes.

File: src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx [hunk 2]
```diff
@@ -16,11 +14,11 @@ export default function CallToActionAlert({
...
   const buttonClass =
     status === 'continue' ? 'va-button-primary' : 'usa-button-primary';
-
-  const alertProps = {
-    headline: heading,
-    level: headerLevel,
-    content: (
+  const CustomHeaderLevel = `h${headerLevel}`;
+  return (
+    // TODO fix this
```
Comment (randimays) [inline src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx:19]: Something still need to be fixed here?

File: src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js [hunk 3]
```diff
@@ -112,11 +110,13 @@ export class CernerCallToAction extends Component {
       });
       return (
         <div data-testid="cerner-cta-widget">
-          <AlertBox
-            headline="Something went wrong"
-            content="We’re sorry. Something went wrong on our end. Please try again later."
-            status="error"
-          />
+          <va-alert status="error">
+            <h3 slot="headline">"Something went wrong"</h3>
```
Comment (randimays) [inline src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js:114]: `Something went wrong` probably should not have quotes.

**Review by randimays — APPROVED** [review]

File: src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx (hunk 1 above)
Comment (jtmst) [reply src/applications/resources-and-support/components/ResourcesAndSupportSearchApp.jsx]: good catch, ty

File: src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js (hunk 3 above)
Comment (jtmst) [reply src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js]: good catch, ty

File: src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx (hunk 2 above)
Comment (jtmst) [reply src/applications/static-pages/cta-widget/components/CallToActionAlert.jsx]: Removed, just forgot to clean up the comment

**Pull request merged**